src/App.tsx

```diff
@@ -1,14 +1,61 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo, TodoWithUser } from './types/Todo';
+import { FILTERS } from './types/FILTERS';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [option, setOption] = useState('all');
+  const [todo, setTodo] = useState<TodoWithUser | null>(null);
+  const [isTodoModal, setIsTodoModal] = useState(false);
+
+  useEffect(() => {
+    getTodos()
+      .then(todosFromServer => {
+        setTodos(todosFromServer);
+      });
+  }, []);
+
+  const onQueryChange = (text: string) => {
```

Comment:
in the parent component - `handle`, in the children component `on`
```suggestion
  const handleQueryChange = (text: string) => {
```